CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.0] - 2021-06-04
+
+- Initial release
```

Comment:
can you move it under the "unreleased section" and update to "initial ruby abstractions release" please?